server/endpoint_detection.py

```diff
@@ -163,7 +164,7 @@ def identify_django_endpoints(self, project_path, project_id):
                                             view_type,
                                         )
                                     )
-                                    print(file_path, identifier)
+                                    logging.info("project_id:" , project_id ,"identify_django_endpoints -> file_path : ",file_path, " and " , "identifier : ",identifier)
```

Comment:
Can we use normal fstrimgs everywhere please

Reply:
done